### PR TITLE
feat(volume_rate): Add cubic_meter_per_min and cubic_meter_per_hour as volume rate units

### PR DESCRIPTION
**Pull request by Uzaaft**
This PR attempts to add  cubic_meter_per_min and cubic_meter_per_hour as volume rate units.

### CODE REVIEW

**Comment by Uzaaft** [comment]
Waiting on #405  before marking this as ready for review.a

File: src/si/volume_rate.rs [hunk 1]
```diff
@@ -35,6 +35,10 @@ quantity! {
             "dam³/s", "cubic decameter per second", "cubic decameters per second";
         @cubic_meter_per_second: prefix!(none); "m³/s", "cubic meter per second",
             "cubic meters per second";
+        @cubic_meter_per_minute: 1.0 / 60.0; "m³/min", "cubic meter per minute",
+            "cubic meters per minute";
+        @cubic_meter_per_hour: 1.0 / 3600.0; "m³/h", "cubic meter per hour",
+            "cubic meters per hour";
```
Comment (iliekturtles) [inline src/si/volume_rate.rs:41]:
```suggestion
        @cubic_meter_per_minute: 1.0_E0 / 6.0_E1; "m³/min", "cubic meter per minute",
            "cubic meters per minute";
        @cubic_meter_per_hour: 1.0_E0 / 3.6_E3; "m³/h", "cubic meter per hour",
            "cubic meters per hour";
```

**Review by iliekturtles — COMMENTED** [review]
Thanks for the PR! Changes look pretty good. Two minor changes below.

**Comment by Uzaaft** [comment]
Thants for the suggestions, implemented those. :)
While I have you here @iliekturtles . I was wondering if you would be open to implement(or open for a contribution) that adds support for standard and normal conditions to the codebase?  
https://en.wikipedia.org/wiki/Standard_temperature_and_pressure

**Comment by iliekturtles** [comment]
Similar to the other PR can you squash the commits and update the commit message. At the same time you can exclude the mass rate changes and just base this on top of iliekturles/master.

I'm definitely open to STP contributions. `AmountOfSubstance` already has a few STP specific units. Are you thinking about adding more or something different to more generally support STP?

**Comment by Uzaaft** [comment]
> Similar to the other PR can you squash the commits and update the commit message. At the same time you can exclude the mass rate changes and just base this on top of iliekturles/master.
> 
> I'm definitely open to STP contributions. `AmountOfSubstance` already has a few STP specific units. Are you thinking about adding more or something different to more generally support STP?

well, what I ideally would want is to have a general support of STP wherever it could apply, as well as adding methods to convert from/into other units. I'll have to take a look at how `AmountOfSubstance` does it.

**Comment by Uzaaft** [comment]
Everything should be ready now.

**Comment by iliekturtles** [comment]
Thanks for the PR!

**Pull request merged**